### ISSUE TITLE
AVT 1 - React Combobox has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3260]
## Environment
macOS Mojave version 10.14.5
Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React

Detailed Description
Run DAP on any of the three Combobox examples and the following DAP violations are present:

<img width="1077" alt="Screen Shot 2019-07-02 at 2 26 14 PM" src="https://user-images.githubusercontent.com/21676914/60541060-5258da80-9cd6-11e9-91ec-004172747e14.png">

**Comment by snidersd** [comment]
This issue is still present when tested with DAP. See the screenshot from today's test. 
<img width="1164" alt="Screen Shot 2019-10-02 at 4 18 18 PM" src="https://user-images.githubusercontent.com/21676914/66082723-5ba14f80-e530-11e9-9a22-0035a7707a8c.png">

**Comment by snidersd** [comment]
@dakahn In addition to the ID reference violation, the Combobox with items as components also has the following violations when the Combobox is expanded:

<img width="1174" alt="Screen Shot 2019-10-03 at 8 53 44 AM" src="https://user-images.githubusercontent.com/21676914/66132888-6228d900-e5bb-11e9-8d86-9947458c0d8e.png">

**Comment by abbeyhrt** [comment]
Hi @snidersd I'll address the ID reference violation! The violations around color contrast aren't actual problems with the Combobox, the story just has the numbers styled to red inline, I can go in there and switch them to a different color but thankfully this isn't a problem somebody would encounter when implementing Combobox
